bigquery_etl/query_scheduling/generate_airflow_dags.py

```diff
@@ -52,12 +54,12 @@ def get_dags(project_id, dags_config):
     """Return all configured DAGs including associated tasks."""
     tasks = []
     dag_collection = DagCollection.from_file(dags_config)
-
     for project_dir in project_dirs(project_id):
         # parse metadata.yaml to retrieve scheduling information
         if os.path.isdir(project_dir):
             for root, dirs, files in os.walk(project_dir):
                 try:
+                    checks_task = None
```

Comment:
```suggestion
```
I don't think this is needed. `checks_task` gets set on line 100 and is only used in that scope